### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zeibura**
More doc updates for v2.0: SQL server in processor groups, dialects clarification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,9 +239,20 @@ The Find All References and Go To Definition functionalities are extended to wor
...
+Use processor groups to link programs with specific dialects, SQL backend settings, and local folders containing copybooks. You define processor groups in a `proc_grps.json` file and associate them with programs in a `pgm_conf.json` file. Create both of these files in a `/.cobolplugin` folder in your workspace root.
+
+In the `proc_grps.json` file: 
 
-In the `proc_grps.json` file, specify your copybook folders in a `libs` array, and specify dialects within a `preprocessor` JSON element which contains the libraries which use that dialect. Use the following format:
+- Specify your copybook folders in a `libs` array.
+- Specify dialects within a `preprocessor` JSON element. This element can contain a `libs` array which specifies folders that contain copybooks which use that dialect. 
+- Specify SQL backend settings within a separate `preprocessor` JSON element with the following parameters: 
+   - `name` (string)
+   Specify **DB2** or **DATACOM**
```
Comment (zimlu02) [inline README.md]: the correct value is: “DB2/DATACOM”

File: README.md [hunk 2]
```diff
@@ -257,6 +270,10 @@ In the `proc_grps.json` file, specify your copybook folders in a `libs` array, a
                     "libs": [
                 "LIB3", "LIB4"
             ]
+                }, 
+                {
+                    "name": "DB2",
```
Comment (zimlu02) [inline README.md]: same here - it should be: “DB2/DATACOM”

**Review by zimlu02 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ishche) [reply README.md]: @VitGottwald will it make sense to rename the preprocessor into "SQL"?

Comment (zeibura) [reply README.md]: That would make a lot more sense

Comment (ishche) [reply README.md]: changed in  https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/1905
It should be `"name": "SQL"` now

**Review by ishche — APPROVED** [review]

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**